econplayground/templates/assignment/assignment_detail.html

```diff
@@ -20,6 +20,12 @@
 
 {% block pagetitle %}
     <div class="float-end">
+        <a class="btn btn-secondary display-inline mx-2"
+           role="button"
+           href="{% url 'assignment_student' object.pk %}"
```

Comment:
The views that these links link to are not used in the current assignment setup - I suppose I need to clean this up.

Reply:
Where would you like me to point them to?

Reply:
For the assignment detail preview, we have the links to each Step, which is essentially student preview. We don't really have "main page" for the assignment on the student side yet, but I'll be putting that together soon.

For previewing what the student sees on the assignment list, that will depend on which courses the student is part of, which is still to be determined, but i'll be working on that surrounding stuff this sprint.

Reply:
Okay, I'll hold off on updating this then